### ISSUE TITLE
CMO not using daily momentum

### ISSUE THREAD

**Issue by mihakralj** [DaveSkender/Stock.Indicators#961]
### the problem

I tested the CMO oscillator on four different TA libraries (TA-LIB, Tulip, Pandas-TA and this one here). The results are stunningly widely different, and no two libs are producing comparable outputs...

One key difference is if library is summarizing up/down DIFFERENCES from previous prices, or summarizing absolute up/down values. I will try to find the actual write-up of CMO by Tushar Chande in his 1994 book _The New Technical Trader_

**Comment by mihakralj** [comment]
Looks like the book defines CMO as a sum of DIFRFERENCES between previous value and today value, not the sum of absolute values:

![image](https://user-images.githubusercontent.com/31756078/206595953-ab02f2ce-514e-4941-a711-76298c174bf4.png)
![image](https://user-images.githubusercontent.com/31756078/206595986-af72ad1e-f8a1-4786-bf6c-b17d3f82aa73.png)

**Comment by mihakralj** [comment]
@twopirllc Pandas-TA is also calculating CMO differently than what Chande's book prescribes:
```
ff = ta.DataFrame({'close': [101.0313, 101.0313, 101.125, 101.9687, 102.7813, 103, 102.9687, 103.0625, 102.9375, 102.7188, 102.75, 102.9063, 102.9687 ]})
qq = ta.cmo(close = ff['close'], length=10)

0        NaN
1        NaN
2        NaN
3        NaN
4        NaN
5        NaN
6        NaN
7        NaN
8        NaN
9        NaN
10    55.221
11    60.091
12    61.928
Name: CMO_10, dtype: float64
```

**Comment by DaveSkender** [comment]
I think this is the same as what I see on [Investopedia](https://www.investopedia.com/terms/c/chandemomentumoscillator.asp).  This isn’t a super complicated one, but will double-check [the code](https://github.com/DaveSkender/Stock.Indicators/blob/35c46e66a15e7d289a4313f127cc53889bcc6ef4/src/a-d/Cmo/Cmo.Series.cs)  to see if there’s any issues.

**Comment by mihakralj** [comment]
Yes, Investopedia is referring the book but then gives the wrong cmo formula explanation...

**Comment by DaveSkender** [comment]
> Yes, Investopedia is referring the book but then gives the wrong cmo formula explanation...

Oooooh.  I see how Investopedia got this wrong.  Definitely taking them off my reputable sources list and sticking to original author publications.

With that said, I’m not quite sure yet if I didn’t account for diff prices, given that the close prices cancel each other out when you simplify the math.

**Comment by DaveSkender** [comment]
this was fixed in v2.4.3
